packages/cli/src/commands/configure/configure_profile_command.test.ts

```diff
@@ -6,14 +6,23 @@ import { ConfigureProfileCommand } from './configure_profile_command.js';
 import { AmplifyPrompter, printer } from '@aws-amplify/cli-core';
 import { Open } from '../open/open.js';
 import { ProfileController } from './profile_controller.js';
+import { UsageDataEmitter } from '@aws-amplify/platform-core';
 
 const testAccessKeyId = 'testAccessKeyId';
 const testSecretAccessKey = 'testSecretAccessKey';
 const testProfile = 'testProfile';
 const testRegion = 'testRegion';
 
+const emitFailureSpy = mock.fn<UsageDataEmitter['emitFailure']>(() =>
+  Promise.resolve()
+);
+const emitSuccessSpy = mock.fn<UsageDataEmitter['emitSuccess']>(() =>
+  Promise.resolve()
+);
```

Comment:
nit: this looks like a mock not spy.

Reply:
renamed to emitFailureMock, type is added there intentionally, so the arguments are typed when I am doing assertions on these

Reply:
I do not see rename effect.